plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/hooks.ts

```diff
@@ -40,10 +47,11 @@ export default ({
   }, [selectedGroup, dataset.components]); // eslint-disable-line react-hooks/exhaustive-deps
 
   useEffect(() => {
-    if (activeComponentIDs) {
-      onOverride?.(dataset.dataID, activeComponentIDs);
+    const buildingSearchActive = buildingSearch?.find(b => b.dataID === dataset.dataID)?.active;
+    if (activeComponentIDs || buildingSearchActive) {
+      onOverride?.(dataset.dataID, [...(activeComponentIDs ? activeComponentIDs : [])]);
```

Comment:
I can't decide if it is good or not, but these days I've been shortening this kind of situation to something like
`...(activeComponentIDs ?? [])`. 
You don't have to change here, but just in case you like shorter code :P

Reply:
Seems better! I updated this.